nc-games/src/Components/NewComment.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import { postComment } from "../api";
+
+function NewComment({reviewId, hasPostedComment, setHasPostedComment, setNumComments, setComments}) {
+    const [newComment, setNewComment] = useState("");
+    const [isPostingError, setIsPostingError] = useState(false)
+    const [hasPosted, setHasPosted] = useState(false)
+    const [hasPostedError, setHasPostedError] = useState(false)
+    const [zeroLength, setZeroLength] = useState(false)
+    const [user, setUser] = useState("grumpy19")
```

Comment:
If you wanted an extra bit - you could add a little login button that logs you in as grumpy. 
Means you could add functionality to stop someone posting if not logged in.
Could later build it out to be a login/user select page